classes/migration/install/FailedJobsMigration.inc.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+/**
+ * @file classes/migration/install/FailedJobsMigration.inc.php
+ *
+ * Copyright (c) 2014-2021 Simon Fraser University
+ * Copyright (c) 2000-2021 John Willinsky
+ * Distributed under the GNU GPL v3. For full terms see the file docs/COPYING.
+ *
+ * @class FailedJobsMigration
+ * @brief Describe database table structures.
+ */
+
+namespace PKP\migration\install;
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class FailedJobsMigration extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up()
+    {
+        if (!Schema::hasTable('failed_jobs')) {
```

Comment:
Is there a reason this condition is present? The other `install` migrations don't check for table existence.

Reply:
This migration file it's used for `install` and `upgrade` flows. I added it to handle scenarios where this table exists already.

Reply:
If the schema in this file ever changes, that'll lead to silent failures where the `hasTable` check skips creating the already-existing table but the SQL expects a different table structure. I'd rather leave out the condition and have it fail than have to debug a missing schema change later.

Reply:
Okay!

Without versioning the migrations ([like default Laravel and his timestamp named migrations did](https://laravel.com/docs/8.x/migrations#generating-migrations)), removing this `hasTable` is the best approach, so far.